piksi_tools/console/settings_view.py

```diff
@@ -566,29 +577,53 @@ def _settings_export_to_file_button_fired(self):
                           default_directory=swift_path,
                           default_filename='config.ini',
                           wildcard='*.ini')
-        is_ok = file.open()
-        if is_ok == OK:
-            print('Exporting settings to local path {0}'.format(file.path))
-            # copy settings so we can modify dict in place to write for configparser
-            settings_out = {}
-            # iterate over nested dict and set inner value to a bare string rather than dict
-            for section in self.settings:
-                settings_out[section] = {}
-                for setting, inner_dict in self.settings[section].iteritems():
-                    settings_out[section][setting] = str(inner_dict.value)
-            # write out with config parser
-            parser = configparser.RawConfigParser()
-            # the optionxform is needed to handle case sensitive settings
-            parser.optionxform = str
-            parser.read_dict(settings_out)
-            # write to ini file
-            try:
-                with open(file.path, "w") as f:
-                    parser.write(f)
-            except IOError as e:
-                print('Unable to export settings to file due to IOError: {}'.format(e))
-        else:  # No error message because user pressed cancel and didn't choose a file
-            pass
+
+        if file.open() != OK:
+            # No error message here because user likely pressed cancel when choosing file
```

Comment:
Is this always true?  Can we check if there was an error opening the file?  Maybe we should print a message here?

Reply:
https://swift-nav.atlassian.net/browse/ESD-987